Formula/sigrok-cli.rb

```diff
@@ -0,0 +1,39 @@
+class SigrokCli < Formula
+  desc "Sigrok command-line interface to use logic analyzers and more"
+  homepage "https://sigrok.org/"
+  license "GPL-3.0-or-later"
+  revision 1
+
+  head "git://sigrok.org/sigrok-cli", branch: "master"
+
+  stable do
+    url "git://sigrok.org/sigrok-cli",
+        tag:      "sigrok-cli-0.7.2",
+        revision: "b584f959edb788f1731d5a304badf241ac21bf65"
+  end
+
+  livecheck do
+    url :head
+    regex(/^sigrok-cli-(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on "libsigrok"
+  depends_on "libsigrokdecode"
+
+  def install
+    system "./autogen.sh"
+    mkdir "build" do
+      system "../configure", *std_configure_args
+      system "make", "install"
+    end
+  end
+
+  test do
+    system "#{bin}/sigrok-cli", "-L"
```

Comment:
Can we check some output here?

Reply:
We could, but it wouldn't be particularly meaningful. `sigrok-cli -L` just lists the supported drivers and decoders. My intention with this test is to make sure that the command line tool is able to use both the driver library and the decoder library (i.e., everything was linked and installed correctly).

The CLI is really just a pretty thin wrapper around libsigrok and libsigrokdecode, so we could add some more sophisticated tests in the libraries instead. For example, we could run the built-in test suites as a part of the build process.

Reply:
> We could, but it wouldn't be particularly meaningful. sigrok-cli -L just lists the supported drivers and decoders.

That's more meaningful then you might realize. If we can check for an expected output format we can avoid things like failing shared libraries accidentally reporting as a success.

Reply:
These libraries aren't loaded at runtime though, so the application wouldn't start in the first place. I should be able to test one of the demo devices that don't require any hardware. One moment.

Reply:
OK, changed the test to capture samples using the test device.